Makefile.am

```diff
@@ -1,302 +0,0 @@
-# Copyright 2013-present Barefoot Networks, Inc.
```

Comment:
I don't understand why do you need to remove Makefile.am again. Have you rebased?